### PR TITLE
[zh] Add "CSS Properties and Values API" translation

### PR DESCRIPTION
**Pull request by gxy5202**
### Description

Add "CSS Properties and Values API" simplified Chinese translation.

### Motivation

There is no zh-CN version for "CSS Properties and Values API".

### Additional details

原文 https://developer.mozilla.org/en-US/docs/Web/API/CSS_Properties_and_Values_API

### CODE REVIEW

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: CSS Properties and Values API
+slug: Web/API/CSS_Properties_and_Values_API
+---
+
+{{DefaultAPISidebar("CSS Properties and Values API")}}
+
+**CSS Properties and Values API** — [CSS Houdini](/en-US/docs/Web/Guide/Houdini) API 的一部分 — 允许开发者显式地定义他们的 {{cssxref('--*', 'CSS custom properties')}}，允许设置属性类型检查、默认值以及是否可继承其值。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:8]:
```suggestion
**CSS Properties and Values API** — [CSS Houdini](/zh-CN/docs/Web/Guide/Houdini) API 的一部分——允许开发者显式地定义他们的 {{cssxref('--*', 'CSS custom properties')}}，允许设置属性类型检查、默认值以及是否可继承其值。
```

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## 规范
+
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参考
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:48]:
```suggestion
## 参见
```

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+- [Using the CSS properties and values API](/en-US/docs/Web/API/CSS_Properties_and_Values_API/guide)
+- [CSS Painting API](/en-US/docs/Web/API/CSS_Painting_API)
+- [CSS Typed Object Model](/en-US/docs/Web/API/CSS_Typed_OM_API)
+- [CSS Houdini](/en-US/docs/Web/Guide/Houdini)
```
Comment (JasonLamv-t) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:53]:
```suggestion
- [Using the CSS properties and values API](/zh-CN/docs/Web/API/CSS_Properties_and_Values_API/guide)
- [CSS Painting API](/zh-CN/docs/Web/API/CSS_Painting_API)
- [CSS Typed Object Model](/zh-CN/docs/Web/API/CSS_Typed_OM_API)
- [CSS Houdini](/zh-CN/docs/Web/Guide/Houdini)
```

**Review by JasonLamv-t — CHANGES_REQUESTED** [review]
Check https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+title: CSS Properties and Values API
+slug: Web/API/CSS_Properties_and_Values_API
+---
+
+{{DefaultAPISidebar("CSS Properties and Values API")}}
+
+**CSS Properties and Values API** — [CSS Houdini](/zh-CN/docs/Web/Guide/Houdini) API 的一部分——允许开发者显式地定义他们的 {{cssxref('--*', 'CSS custom properties')}}，允许设置属性类型检查、默认值以及是否可继承其值。
```
Comment (yin1999) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:8]:
```suggestion
**CSS 属性和值 API**（CSS Properties and Values API）——[CSS Houdini](/zh-CN/docs/Web/Guide/Houdini) API 的一部分——允许开发者显式地定义它们的 {{cssxref('--*', 'CSS custom properties')}}，允许设置属性类型检查、默认值以及是否可继承其值。
```

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+## 接口
+
+- {{domxref('CSS.registerProperty')}}
+  - : 定义了浏览器应该如何解析 {{cssxref('--*', 'CSS custom properties')}}。通过 [JavaScript](/zh-CN/docs/Web/JavaScript) 中的 {{domxref('CSS.registerProperty')}} 访问该接口。
+- {{cssxref('@property')}}
+  - : 定义了浏览器应该如何解析 {{cssxref('--*', 'CSS custom properties')}}。通过 [CSS](/zh-CN/docs/Web/CSS) 中的 {{cssxref('@property')}} [at-rule](/zh-CN/docs/Web/CSS/At-rule) 访问该接口。
```
Comment (yin1999) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:15]:
```suggestion
  - : 定义了浏览器应该如何解析 {{cssxref('--*', 'CSS custom properties')}}。通过 [CSS](/zh-CN/docs/Web/CSS) 中的 {{cssxref('@property')}} [at 规则](/zh-CN/docs/Web/CSS/At-rule)访问该接口。
```

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
+---
+title: CSS Properties and Values API
```
Comment (yin1999) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:2]:
```suggestion
title: CSS 属性和值 API
```

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Using the CSS properties and values API](/zh-CN/docs/Web/API/CSS_Properties_and_Values_API/guide)
+- [CSS Painting API](/zh-CN/docs/Web/API/CSS_Painting_API)
+- [CSS Typed Object Model](/zh-CN/docs/Web/API/CSS_Typed_OM_API)
```
Comment (yin1999) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:52]:
```suggestion
- [CSS 类型对象模型](/zh-CN/docs/Web/API/CSS_Typed_OM_API)
```

**Comment by gxy5202** [comment]
> Check https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

Thanks

**Review by yin1999 — APPROVED** [review]
Thanks

File: files/zh-cn/web/api/css_properties_and_values_api/index.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+## 接口
+
+- {{domxref('CSS.registerProperty')}}
+  - : 定义了浏览器应该如何解析 {{cssxref('--*', 'CSS custom properties')}}。通过 [JavaScript](/zh-CN/docs/Web/JavaScript) 中的 {{domxref('CSS.registerProperty')}} 访问该接口。
+- {{cssxref('@property')}}
+  - : 定义了浏览器应该如何解析 {{cssxref('--*', 'CSS custom properties')}}。通过 [CSS](/zh-CN/docs/Web/CSS) 中的 {{cssxref('@property')}} [at 规则](/zh-CN/docs/Web/CSS/At-rule)访问该接口。
+
+## 示例
+
+以下代码在 [JavaScript](/zh-CN/docs/Web/JavaScript) 中使用 {{domxref('CSS.registerProperty')}} 输入一个 {{cssxref('--*', 'CSS custom properties')}}，`--my-color`，将其作为一个颜色，然后为其设置一个默认值，并且不允许继承它的值。
```
Comment (yin1999) [inline files/zh-cn/web/api/css_properties_and_values_api/index.md:19]:
```suggestion
以下代码在 [JavaScript](/zh-CN/docs/Web/JavaScript) 中使用 {{domxref('CSS.registerProperty')}} 输入一个 {{cssxref('--*', 'CSS 自定义属性')}}，`--my-color`，将其作为一个颜色，然后为其设置一个默认值，并且不允许继承它的值。
```

**Pull request merged**